themes/digital.gov/layouts/partials/core/collection-list-item.html

```diff
@@ -0,0 +1,69 @@
+{{/*
+
+  Collection List Item
+  Returns a list item variant for USWDS Collection
+
+  @see see collection.html for description of other variables
+  @link_external {boolean} apply usa-link--external class if url host is not www.digital.gov
+
+*/}}
+{{ $variant := .variant }}
+{{ $meta_source := .meta_source }}
+{{ $href := .href }}
+{{ $link_external := false }}
+{{ $logo_source := .logo_source }}
+{{ $list_header := .list_header }}
+{{ $url := urls.Parse $href }}
+
+{{/*  Check if external link  */}}
+{{ if ne $url.Host "www.digital.gov" }}
+  {{ $link_external = true }}
+{{ end }}
+
+{{- with .item -}}
+  {{ $path := "" }}{{ with .File }}
+    {{ $path = .Path }}
+  {{ else }}
+    {{ $path = .Path }}
+  {{ end }}
+
+
+  <li class="usa-collection__item" data-edit-this="{{- $path -}}">
+    {{ if eq $variant "media" }}
+      {{- partial "partials/core/collection-icon" (dict "item" . "logo_source" $logo_source) -}}
+    {{ end }}
+    <div class="usa-collection__body">
+      <h{{ $list_header | safeHTML }} class="usa-collection__header">
+        <a class="usa-link {{ if $link_external }}usa-link--external{{ end }}" href="{{- $href -}}" title="{{- .Title | markdownify -}}">
+          {{- .Title | markdownify -}}
+        </a>
+      </h{{ $list_header | safeHTML }}>
+      {{ if or (eq $variant "default") (eq $variant "media") }}
+        <p class="usa-collection__description">
+          {{- if .Params.summary -}}
+            {{- .Params.summary | markdownify | truncate 140 -}}
+          {{- else -}}
+            {{- .Params.deck | markdownify -}}
+          {{- end -}}
+        </p>
+      {{ else if eq $variant "headings" }}
+        {{ if $meta_source }}
+          <ul class="usa-collection__meta" aria-label="More information">
```

Comment:
It's unclear what there's more information about. This might also cause an a11y issue with duplicate aria-labels.

Reply:
Good point, I've removed the `aria-label`.